plugins/GSdx/GSSetting.cpp

```diff
@@ -102,9 +102,6 @@ const char* dialog_message(int ID, bool* updateText) {
 				"High:\nExtend it to destination alpha blending and color wrapping. (help shadow and fog effect). A good CPU is required.\n\n"
 				"Full:\nExcept few cases, the blending unit will be fully emulated by the shader. It is ultra slow! It is intended for debug.\n\n"
 				"Ultra:\nThe blending unit will be completely emulated by the shader. It is ultra slow! It is intended for debug.";
-		case IDC_SAFE_FBMASK:
```

Comment:
Maybe it's enum declaration could also be removed from ``GSSetting.h`` ? (Linux only)